### PR TITLE
renamed pdcli to kite

### PR DESCRIPTION
**Pull request by supreeth7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,34 +54,34 @@ $ make build
 To log into PagerDuty CLI use the command:
 
 ```
-pdcli login
+kite login
 ```
 This will prompt the user for an API key with necessary instructions of how to generate one. The API key will be saved for future use to the `~/.config/pagerduty-cli/config.json` file.
```
Comment (T0MASD) [inline README.md:59]: should the config path be updated? i.e. ~/.config/kite vs ~/.config/pagerduty-cli

Comment (supreeth7) [reply README.md:59]: It can be, updating now

Comment (T0MASD) [reply README.md:59]: thanks for updating the source, the readme still has old reference tho

Comment (supreeth7) [reply README.md]: Sorry about that, pushed the changes

**Review by T0MASD — APPROVED** [review]
/approved
/lgtm
/label tide/merge-method-squash

**Pull request merged**